todo_app/app.py

```diff
@@ -8,8 +15,40 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    print("index()")
+    items = sorted(get_items(), key=lambda item: item["status"], reverse=True)
+    for i in get_items():
+        print(i)
+    return render_template('index.html', items=items) 
 
+@app.route('/items', methods=['POST'])
+def add_new_item():
+    print("add_new_item")
+    title = request.form['title']
+    print(title)
+    add_item(title)
+    # HELP: how to redirect back to index
+    return index()
```

Comment:
If you change `from werkzeug.utils import redirect` to `from flask import redirect` at the start of the file then you can do `return redirect('/')`

Reply:
thanks!